ipad.css

```diff
@@ -0,0 +1,169 @@
+html, body, div, span, applet, object, iframe,
```

Comment:
when it comes to media queries, you don't need to copy over the css from your base css file. so in this case, your main.css is your base css. this means that when your 'ipad.css' media query takes effect, all the styling from your main.css file will be carried over. if there is a styling conflict, your ipad.css styling will take precedence.